classes/site/Version.inc.php

```diff
@@ -284,6 +285,47 @@ function getVersionString($numeric = true) {
 
 		return $numericVersion;
 	}
+
+	/**
+	 * Checks if this version is compatible with given version
```

Comment:
We are overloading the meaning of "version" here: both to mean a specific version (or the `Version` object) and to mean a version constraint string.  This should be clarified.

Reply:
`Checks if the Version is compatible with the given version constraint string
formatted per composer/semver specifications; 
c.f. https://getcomposer.org/doc/articles/versions.md#writing-version-constraints` 
I liked the included link to the specifications.